src/templates/base/index.html

```diff
@@ -1,8 +1,8 @@
 {% extends "%s/%s/base.html" % (lang, year) %}
 
-{% block image_url %}https://almanac.httparchive.org/static/images/home-hero-bg.png{% endblock %}
-{% block image_height %}562{% endblock %}
-{% block image_width %}715{% endblock %}
+{% block image_url %}https://almanac.httparchive.org{{ get_versioned_filename('/static/images/home-hero-' + year + '.png') }}{% endblock %}
+{% block image_height %}600{% endblock %}
+{% block image_width %}1200{% endblock %}
```

Comment:
Height and Width are different for 2020 - can we make it the same?

```
% identify -format '%w %h %b %i\n' home*.png 
1200 600 148774B home-hero-2019.png
960 540 126188B home-hero-2020.png
1200 600 141273B home-hero-bg.png
820 562 38042B home-hero.png
```

Reply:
Good catch! Done.